docker-compose-installed.yaml

```diff
@@ -46,7 +46,7 @@ services:
         "
 
   rest-api:
-    image: hyperledger/sawtooth-rest-api:1.1
+    image: hyperledger/sawtooth-rest-api:1.2
```

Comment:
Should we use `:latest` here so this always uses the last release?

Reply:
Maybe that should be part of a separate commit